### DISCUSSION TITLE
Invariant Violation: "taroDemo" has not been registered.

### DISCUSSION THREAD

**Discussion by lynnclub** [NervJS/taro#12923] [Q & A] [answered]
BUNDLE  ./index

LOG  Running "taroDemo" with {"rootTag":11}
ERROR  Invariant Violation: "taroDemo" has not been registered. This can happen if:

* Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
* A module failed to load due to an error and `AppRegistry.registerComponent` wasn't called.

**Comment by zhiqingchen** [comment, marked as answer]
https://docs.taro.zone/docs/react-native#invariant-violation-tarodemo-has-not-been-registered
